collaboration.html

```diff
@@ -0,0 +1,171 @@
+		<link href="https://fonts.googleapis.com/css?family=Fjalla+One" rel="stylesheet"> 
+
+
+		<!-- external CSS link -->
+		<link rel="stylesheet" href="css/reset.css">
+		<link rel="stylesheet" href="css/style.css">
+		
+	</head>
+	
+	<body>
+		<div class="container">
+		<header class="clearfix">
+				<div class="logo">
+					<a href="index.html"><img src="images/logo.png" alt="DRRP Transition to Employment logo"></a>
+				</div>
+				<nav>
+					<ul>
+						<li><a href="aboutus.html">ABOUT US</a>
+							<ul>
+								<li><a href="studies.html">Studies</a></li>
+								<li><a href="partners.html">Partners</a></li>
+								<li><a href="advisors.html">Advisors</a></li>
+							</ul>
+						</li>
+						<li><a href="toolkit.html">TOOLKIT</a>
+							<ul class="navalign">
+								<li><a href="collaboration.html">Collaboration</a></li>
+								<li><a href="employment.html">Employment</a></li>
+								<li><a href="intervention.html">Intervention</a></li>
+								<li><a href="skilledprofessional.html">Skilled Profesional</a></li>
+							</ul>
+						</li>
+						<li><a href="contactus.html">CONTACT</a></li>
+					</ul>
+				</nav>
+		</header>
+
+
+		<section>
+			<div class="mainphoto collaboration">
+				<h1 class="homepageheader">COLLABORATION</h1>
+			</div>
+			&nbsp;
+			<div>
+				<p>Collaboration among agencies is necessary for students and youth who are touched by many systems and professionals. When collaboration is directly focused on outcomes for youth, such as integrated paid employment, and the systems that serve them – rather than merely making a hand-off to the next responsible organization – higher rates of employment are more likely. Conversely, without a focus on developing collaborative relationships efforts falter.</p>
+					<br>
+				<div class="container">
+					<h2>What is it?<span class="expand"><a href="#"> +</a></span></h2>
+					<p class="advisory content">Collaboration between schools, vocational rehabilitation (VR), and other partners is effective when there is a clear and compelling rationale for staff to work across agency lines (Fabian, in press). Collaboration is structured around formalized relationships and processes that maximize the expertise and perspectives of students, parents, educators, VR counselors, and other partners involved in the transition process. The relationships and processes should promote individualized student services, supports, and activities and roles, and identify responsibilities of each partner designed and focused on providing the identified needs and supports. The most effective teams work to achieve a direct outcome for youth served, rather than simply coordinate a “hand off” to the next available post-secondary service (Simonsen, Fabian, &amp; Luecking, in press).</p>
+					<p class="collapse"><a href="#" class="readless">Read less &gt;</a><hr></p>
+				</div>
+				
+				<div class="container">
+					<h2>Why is it important?<span class="expand"><a href="#"> +</a></span></h2>
```

Comment:
Do you need the extra `span` tag around the `a` tag? could it just be

```html
<a href="#" class="expand"> +</a>
```
Obviously you'll need to update your JavaScript that hooks into this to remove the `span` as well